nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestDistributionObject.java

```diff
@@ -65,12 +71,45 @@ public void setup() {
 
     @Test
     public void testDistribution() throws IOException {
+        final String PRODUCT_NAME = "cool product name";
+        final String PRODUCT_VERSION = "cool product version";
+        final String DISTRIBUTION_NAME = "cool distribution name";
+        final String DISTRIBUTION_VERSION = "cool distribution version";
+        final String DISTRIBUTION_SERVER = "cool distribution server";
+        final String DISTRIBUTION_DATE = "cool distribution date";
+
+        Map<String, String> testProps = new HashMap<>();
+        testProps.put(Environment.PRODUCT_NAME, PRODUCT_NAME);
+        testProps.put(Environment.PRODUCT_VERSION, PRODUCT_VERSION);
+        testProps.put(Environment.DISTRIBUTION_NAME, DISTRIBUTION_NAME);
+        testProps.put(Environment.DISTRIBUTION_VERSION, DISTRIBUTION_VERSION);
+        testProps.put(Environment.DISTRIBUTION_SERVER, DISTRIBUTION_SERVER);
+        testProps.put(Environment.DISTRIBUTION_DATE, DISTRIBUTION_DATE);
+        Framework.getProperties().putAll(testProps);
+
         try (CloseableClientResponse response = get()) {
             assertEquals(200, response.getStatus());
             JsonNode node = mapper.readTree(response.getEntityInputStream());
-            JsonNode foo = node.get("foo");
-            assertNotNull(foo);
-            assertEquals("bar", foo.asText());
+            JsonNode value = node.get(Environment.PRODUCT_NAME);
+            assertEquals(PRODUCT_NAME, value.asText());
+            value = node.get(Environment.PRODUCT_VERSION);
+            assertEquals(PRODUCT_VERSION, value.asText());
+            value = node.get(Environment.DISTRIBUTION_NAME);
+            assertEquals(DISTRIBUTION_NAME, value.asText());
+            value = node.get(Environment.DISTRIBUTION_VERSION);
+            assertEquals(DISTRIBUTION_VERSION, value.asText());
+            value = node.get(Environment.DISTRIBUTION_SERVER);
+            assertEquals(DISTRIBUTION_SERVER, value.asText());
+            value = node.get(Environment.DISTRIBUTION_DATE);
+            assertEquals(DISTRIBUTION_DATE, value.asText());
+            value = node.get(Environment.WARNINGS);
+            assertNotNull(value);
+            value = node.get(Environment.BUNDLES);
+            assertNotNull(value);
+            String flat = value.toString();
+            JsonAssert.on(flat).get(0).get("name").notNull();
```

Comment:
You should also test "version" to validate the list item structure.
Something like:
```
JsonAssert jsonAssert = JsonAssert.on(flat).isArray().get(0);
jsonAssert.get("name").notNull();
jsonAssert.get("version").notNull();
```